app/controllers/admin/communities_controller.rb

```diff
@@ -189,13 +189,20 @@ def settings
           [SettingsViewUtils.search_type_translation(type), type]
         }
 
+      distance_unit_select_options = [
+          [SettingsViewUtils.distance_unit_translation(:km), :metric],
+          [SettingsViewUtils.distance_unit_translation(:miles), :imperial]
+      ]
+
       render :settings, locals: {
         supports_escrow: escrow_payments?(@current_community),
         delete_redirect_url: delete_redirect_url(APP_CONFIG),
         delete_confirmation: @current_community.ident,
         can_delete_marketplace: can_delete_marketplace?(@current_community.id),
         main_search: marketplace_configurations[:main_search],
-        main_search_select_options: main_search_select_options
+        main_search_select_options: main_search_select_options,
+        distance_unit: marketplace_configurations[:distance_unit],
+        distance_unit_select_options: distance_unit_select_options
```

Comment:
Is this the usual way to give dropdown options? To me it looks like it could be done in HAML with regular translations